core/src/main/scala/com/softwaremill/session/SessionConfig.scala

```diff
@@ -112,6 +112,6 @@ object SessionConfig {
   /**
    * Creates a default configuration using the given secret.
    */
-  def default(serverSecret: String) = fromConfig(ConfigFactory.load()
+  def defaultConfig(serverSecret: String) = fromConfig(ConfigFactory.load()
```

Comment:
maybe we can keep the `default` method, and add another one, `defaultConfig`, which would just delegate (or the other way round). We won't break exisitng cod this way